modules/cnv-vm-wizard-fields-web.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/TBD
+[id="cnv-vm-wizard-fields-web_{context}"]
+== Virtual machine wizard fields
+
+|===
+|Name |Parameter |Description
+
+|Name
+|
+|Name of the virtual machine. The name can contain lower-case letters (`a-z`), numbers (`0-9`), periods (`.`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name may not contain spaces or special characters.
+
+|Description
+|
+|Optional description field.
+
+|Template
+|
+|Template from which to create the virtual machine. Selecting a template will automatically complete other fields.
+
+.5+|Provision Source
+|PXE
+|Provision virtual machine from PXE menu. Requires a xref:pxebooting[PXE-capable NIC] in the cluster.
```

Comment:
no xrefs in modules

Reply:
Please add appropriate articles to these entries if you keep them..

Provision *a* virtual machine from *the* ...

Reply:
You need to remove the xrefs.